lib/Runtime/PlatformAgnostic/Platform/Windows/ArraysUtility.cpp

```diff
@@ -0,0 +1,41 @@
+//-------------------------------------------------------------------------------------------------------
+// Copyright (C) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+//-------------------------------------------------------------------------------------------------------
+
+#include "RuntimePlatformAgnosticPch.h"
+#include "Common.h"
+#include "ChakraPlatform.h"
+
+namespace PlatformAgnostic
+{
+namespace Arrays
+{
+
+    size_t Utility::GetLocaleSeparator(WCHAR* szSeparator)
+    {
+
+        LCID lcid = GetUserDefaultLCID();
```

Comment:
Nit: we have been trying to move away from LCIDs since we don't require Vista support, so `GetUserDefaultLocaleName` and `GetLocaleInfoEx` are probably preferred.